### PR TITLE
Feature/main palette card

### PR DESCRIPTION
**Pull request by r1g023**
# Description
Issue # (add the issue number here, if applicable)

Describe what you're adding
- added the main card where users will be able to see the recommended options
- see attached
- main card

# QA
List out how to test your changes and add a screenshot of what it should look like, when applicable.

### CODE REVIEW

File: src/components/PaletteMainCard.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import { Box } from "@chakra-ui/react";
+
+function PaletteMainCard() {
+  return (
+    <Box gridArea="main" bg="white.500" borderLeft={"1px solid gray"} p={4}>
+      <h1>Palette #200123</h1>
```
Comment (zeeberry) [inline src/components/PaletteMainCard.js:6]: small nit. we can use chakra heading here.

File: src/components/LandingPage.js [hunk 2]
```diff
@@ -24,6 +25,8 @@ function LandingPage() {
       <LeftUpperCard />
       {/* Left Bottom Card */}
       <LeftBottomCard />
+      {/* Palette Main Card */}
```
Comment (zeeberry) [inline src/components/LandingPage.js:28]: same comment as previous PRs. unnecessary comment here.

**Review by zeeberry — COMMENTED** [review]
Small comments. Looks good.

File: src/components/PaletteMainCard.js (hunk 1 above)
Comment (r1g023) [reply src/components/PaletteMainCard.js]: Added <Heading> and removed H1 tag
@zeeberry

File: src/components/LandingPage.js (hunk 2 above)
Comment (r1g023) [reply src/components/LandingPage.js]: Apologies, removed the comments on the LandingPage component, pushed changes to 'feature/main-palette-card' branch

**Review by zeeberry — APPROVED** [review]

**Pull request merged**